include/GafferUI/Nodule.h

```diff
@@ -66,8 +66,6 @@ class Nodule : public ConnectionCreator
 		const Gaffer::Plug *plug() const;
 
 		void updateDragEndPoint( const Imath::V3f position, const Imath::V3f &tangent ) override;
-		bool canCreateConnection( const GafferUI::Nodule *endpoint ) override;
```

Comment:
Let's do the same with `updateDragEndPoint()` as well, since it's part of the same API.